rust/ddlog_benches/benches/twitter.rs

```diff
@@ -45,6 +49,52 @@ fn twitter_micro(c: &mut Criterion) {
     }
 }
 
+fn twitter_micro_with_profiler(c: &mut Criterion) {
+    let mut group = c.benchmark_group("twitter-micro-with-profiler");
+    group.sampling_mode(SamplingMode::Flat);
+    group.sample_size(10);
+
+    for record_count in record_counts() {
+        let dataset = twitter::dataset(Some(record_count));
+
+        for thread_count in DDLOG_WORKERS.rev() {
+            group.bench_with_input(
+                BenchmarkId::new(
+                    format!(
+                        "{} thread{}",
+                        thread_count,
+                        if thread_count == 1 { "" } else { "s" },
+                    ),
+                    format!("{} records", record_count),
+                ),
+                &dataset,
+                |b, dataset| {
+                    b.iter_batched(
+                        || {
+                            (
+                                twitter::init_config(Config {
+                                    num_timely_workers: thread_count,
+                                    profiling_config: ProfilingConfig::TimelyProfiling {
+                                        timely_destination: LoggingDestination::Disk {
+                                            directory: "timely_trace".to_string(),
+                                        },
+                                        timely_progress_destination: None,
+                                        differential_destination: None,
```

Comment:
Probably should add one that includes ddflow

Reply:
Good point.  differential logs are usually much smaller, so I'll just enable both.